main.py

```diff
@@ -52,11 +51,14 @@ def reAward(fname):
 
 def rePhone(fname):
   #open file
+  fhand = open(fname, 'r')
   #loop through the file
+  for line in fhand:
   #extract the specific phone numbers regex
+    extr = re.findall('902.*', line)
```

Comment:
Consider the pattern for all phone numbers. ###-###-####
`extr = re.findall('\d{3}-\d{3}-\d{4}', line)`